src/antispoof.c

```diff
@@ -144,21 +213,22 @@ static int antispoof_burst(struct pg_brick *brick, enum pg_side from,
 		}
 
 		/* ARP antispoof */
-		if ((eth->ether_type == htobe16(ETHER_TYPE_ARP)) && state->ip) {
-			struct pg_antispoof_arp *a;
-
-			a = (struct pg_antispoof_arp *)(eth + 1);
-			if (antispoof_arp(state, a) < 0) {
+		if (state->arp_enabled) {
+			if (eth->ether_type ==
```

Comment:
can we have something like
```
if ((eth->ether_type == rte_cpu_to_be_16(ETHER_TYPE_ARP) && antispoof_arp(state, eth + 1) < 0) ||
    eth->ether_type == rte_cpu_to_be_16(ETHER_TYPE_RARP))) {
```

 also, you could use a temporary variable for eth->ether_type, and I would use unlikely on this condition, as most packets won't be arp, it's a good deal to accelerate all non arp packets